docs/en/glossary/glossary.asciidoc

```diff
@@ -370,6 +386,10 @@ different sources.
 include::{kib-repo-dir}/glossary.asciidoc[tag=ems-def]
 --
 
+[[glossary-epr]] Elastic Package Registry (EPR)::
```

Comment:
Is it common to include links? https://github.com/elastic/package-registry

Reply:
Sure is. Good idea.